File: WalkthroughClientScripts/Startup.cs
using Bridge;
using System;
using System.Linq;
using UI5;

namespace WalkthroughClientScripts
{
    [Reflectable(false)]
    [FileName("startup.js")]
    [Name("sap.ui.demo.walkthrough.startup")]
    public class Startup
    {
        [Init(InitPosition.Top)]
        public static void StartupScript()
        {
            sap.ui.getCore().attachInit(() => {
                new sap.m.Shell(new sap.m.Shell.Settings() {
                    app = new sap.ui.core.ComponentContainer(new sap.ui.core.ComponentContainer.Settings() {
                        name = "sap.ui.demo.walkthrough",
                        settings = new sap.ui.core.UIComponent.Settings() {
                            id = "walkthrough"
                        }
                    })
                }).placeAt("content");
            });
        }
    }
}

File: walkthrough/wwwroot/bridge/startup.js
sap.ui.getCore().attachInit(function () {
    new sap.m.Shell({ app: new sap.ui.core.ComponentContainer({ name: "sap.ui.demo.walkthrough", settings: { id: "walkthrough" } }) }).placeAt("content");
});

Bridge.assembly("WalkthroughClientScripts", function ($asm, globals) {
    "use strict";

    Bridge.define("sap.ui.demo.walkthrough.startup");
});
